includes/Admin/Settings.php

```diff
@@ -750,4 +751,23 @@ public function refresh_admin_settings_field_options() {
             $this->send_response_error( $e );
         }
     }
+
+    /**
+     * Dokan data clear setting
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @return array $settings_fields
+     */
+    public function add_dokan_data_clear_setting( $settings_fields ) {
+        $settings_fields['data_clear_on_uninstall'] = [
+            'name'    => 'data_clear_on_uninstall',
+            'label'   => __( 'Data Clear', 'dokan-lite' ),
+            'desc'    => __( 'Clear all the data related to Dokan Lite and Pro after plugin deletion.', 'dokan-lite' ),
```

Comment:
kindly change the text with: Delete all data and tables related to Dokan and Dokan Pro plugin while deleting the Dokan plugin.